worker/buildbot_worker/test/unit/test_scripts_base.py

```diff
@@ -33,7 +34,10 @@ class TestIsWorkerDir(misc.FileIOMixin, misc.StdoutAssertionsMixin,
 
     def setUp(self):
         # capture output to stdout
-        self.mocked_stdout = io.BytesIO()
+        if PY3:
```

Comment:
Yes, but when I tried to use twisted.python.compat.NativeStringIO inside
buildbot_worker, tests broke because apparently
you are testing buildbot_worker in environments with old versions of Twisted which do not have NativeStringIO.